output-tests/content/draft-next/general.json

```diff
@@ -0,0 +1,43 @@
+[
```

Comment:
Again ignore if needed but did you consider inverting the directory structure to `output-tests/draft-next/{content,structure}` rather than `output-tests/{content,structure}/draft-next`? They seem equally easy for implementers to run, but I think the former is mildly easier for us to maintain (requires remembering to `cp -R` only one additional directory when a new draft is released)

Which on the latter reminds me, can you add a step to #571's description when we merge this to remember to include updating these too.

Reply:
I can do that.

Reply:
Does that issue just remain perpetually open?

Reply:
No it was/is meant to go in some sort of "documentation site" for the suite (rather than continuing to pile into the README which was meant more for implementers), but the former is still waiting for the `json-schema.org` rewrite/merge...

Obviously up for ideas if you have em on where to put it instead (including in the README if you think we should "give up" for now...)

Reply:
updated